ssm_endpoint_sg.tf

```diff
@@ -0,0 +1,23 @@
+# Security group for the SSM interface endpoint (and other endpoints
+# required when using the SSM service) in the private subnet
+resource "aws_security_group" "ssm_endpoint" {
+  provider = aws.sharedservicesprovisionaccount
+
+  vpc_id = aws_vpc.the_vpc.id
+
+  tags = {
+    Name = "SSM endpoints"
```

Comment:
Is there a reason the resource name is singular but the tag name is plural?

Reply:
That happened because there is a single VPC endpoint service that is attached to multiple subnets, each in its own availability zone.  I can change the tags to use the singular.

Reply:
Please see commit fd19f41.